### ISSUE TITLE
libvirt fails when redirecting a usb device to the vm

### ISSUE THREAD

**Issue by symphorien** [NixOS/nixpkgs#106594]
**Describe the bug**
when redirecting a usb device to a vm, a permission error is raised when running setfacl

this is due to a missing setcap wrapper for spice-client-glib-usb-acl-helper

**To Reproduce**
1. Install libvirt:
```nix
{ config, pkgs, lib, ... }: {
  config = {
    virtualisation.libvirtd = {
      enable = true;
      qemuRunAsRoot = false;
      onBoot = "ignore";
    };
    users.users.symphorien.extraGroups = [ "libvirtd" ];
    environment.systemPackages = [ pkgs.virt-manager pkgs.virt-viewer ];
  };
}
```
2. create a vm in `qemu:///system`, start it
3. redirect any usb device to it
4. Error setting facl: operation not permitted

**Expected behavior**
ticking the checkbox works

**Screenshots**
![image](https://user-images.githubusercontent.com/12595971/101825164-2e353600-3b25-11eb-815d-1a187ccb8d75.png)

**Additional context**
I'll open a PR shortly

**Notify maintainers**
cc @fpletz @globin @xeji maintainers of spice and libvirt

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.8.18, NixOS, 20.09.git.a9085440e57M (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.9`
 - channels(root): `"nixos-20.09.2016.19db3e5ea27, nixos-hardware, nixos-unstable-21.03pre246543.24c9b05ac53"`
 - channels(symphorien): `"home-manager-20.09"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module: libvirt spice
```

**Comment by symphorien** [comment]
A possible solution is implemented in #106595

**Comment by wucke13** [comment]
FFR: setting [`virtualisation.spiceUSBRedirection.enable`](https://search.nixos.org/options?channel=21.11&show=virtualisation.spiceUSBRedirection.enable&from=0&size=50&sort=relevance&type=packages&query=virtualisation.spiceUSBRedirection) to `true` should fix this
